### ISSUE TITLE
Security considerations should talk about public key distribution

### ISSUE THREAD

**Issue by jyasskin** [w3c-ccg/http-signatures#16]
https://web-payments.org/specs/source/http-signatures-audit/#https says that
> It is also possible to mitigate an attack where the Certificate Authority is compromised by digitally signing all information that is intended for the client and the server.

However, that requires [`keyId`](https://tools.ietf.org/id/draft-cavage-http-signatures-07.html#rfc.section.2.1.1) to identify a key that's trusted through some other route than the Web PKI, which means it needs to have been received in a trusted way. I see that specifying key distribution is out of scope for draft-cavage-http-signatures, but people using the spec need to be aware that it's a difficult problem.

**Comment by msporny** [comment]
Agreed. I will add a section explaining that key distribution is out of scope and it is a difficult problem where a solution design and implementation should not be undertaken lightly.

We are working on specs that do new forms of key distribution in specs like https://w3c-ccg.github.io/did-spec/ ... but again, super experimental work and we expect many organizations to implement (or build on top of) their own closed key management systems.

**Comment by liamdennehy** [comment]
@jyasskin @msporny Should this move to https://github.com/w3c-dvcg/http-signatures-audit/issues?
